docs/mine-hnt/validators/requirements.mdx

```diff
@@ -13,23 +13,26 @@ Based on early tests, validator node requirements include:
 - Memory: 8 GB or above
-- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet. This should not be on the OS disk and remoting your `/home/*` directory on a seperate disk is recommended.
-- If using AWS, a t2.xlarge or c5a.xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with similar capabilities should be the target.  
+- Storage: 256 GB or more for mainnet.  64GB is sufficient for testnet. It is recommended that you keep your validator data on a separate disk.
+- If using AWS, a t2.xlarge or c5a.xlarge EC2 instance or equivalent is recommended. If using another cloud provider, like GCP, Azure, or Digital Ocean, an instance with similar capabilities should be the target.
 - Static IP address with ports `2154`(TCP) and `8080`(TCP) opened in your firewall with a route available to your validator host.
 - Running on stable network connection free of things like proxies, NAT, etc. The load is largely symmetrical when producing blocks, so good upstream connectivity recommended.
 - It is [discouraged to run Validators at home](/mine-hnt/validators/faqs-resources-providers/#technical). The one exception to this is if you live at an actual data center. If that's the case, you're allowed to run Validators at home.
 
 - Overall Baselining (do this after the validator is caught up to the tip of the chain, and running normally for a few hours)
-  - Review the Underlying Hardware your provider is using You can review your instance cpu speeds by using `cat /proc/cpuinfo | egrep  -i "name|stepping|Mhz" | sort | uniq -c` Some providers use the Turbo speeds as marketing and others the regular speed of the processor.
+  - Review the underlying hardware your provider is using. You can review your instance cpu speeds by using `cat /proc/cpuinfo | egrep  -i "name|stepping|Mhz" | sort | uniq -c`.  Some providers use the Turbo speeds as marketing and others the regular speed of the processor.
+  - Install sar package using `sudo apt install sysstat`
+    - Confirm that sar is enabled in `/etc/default/sysstat` review that the `ENABLED` line is set to `true`.
+    - Review the cron job and is set to the correct schedule that you would like to collect stats on. You can `sudo vi etc/cron.d/sysstat` file to adjust the default of collecting every 10 minutes `*/10 ...`.
   - You can review your CPU steal using `sar` and looking at the `%steal` column. Averages of over 1% you should think about moving providers. You can also see this metric in `top` look for the `st` cpu metric on the second line.
   - You can review your IOPS using `sar -b` 
-  - You can review your Disk Bandwidth using `sar -d -p | egrep "sd[a-z]|DEV"`
+  - You can review your Disk Bandwidth using `sar -d -p | egrep "sd[a-z]|DEV"` NOTE: this command line is setup for using `/dev/sd<some letter>` device names if your hardware is different you may need to change this commnd as needed.
```

Comment:
> NOTE: this command line is setup for using `/dev/sd<some letter>` device names if your hardware is different you may need to change this commnd as needed.

Nit: Still a bit awkward and has a spelling mistake. How about: 
> NOTE: this example command line is assumes `/dev/sd<some letter>` device names. Your instance may have disks with different device names, so change this command as needed.

Reply:
Sure sounds better to me.